docs/java/java-project.md

```diff
@@ -79,6 +61,22 @@ If you have multiple subfolders that have source code for your project, you can
   <source src="/docs/java/java-project/multiple-source.mp4" type="video/mp4">
 </video>
 
+## Lightweight Mode
+
+VS Code for Java supports two modes, lightweight and standard. With lightweight mode, only source files and JDK are resolved by language server; with standard mode, imported dependencies are resolved and project is built by language server. Lightweight mode works best when you need a super quick-to-start and lightweight environment to work with your source files, for example, reading source code, navigating among source code and JDK, viewing outline and Javadoc, and detecting and fixing syntax errors. Also, code completion is supported within the scope of source files and JDK.
+
+Lightweight mode does neither resolve imported dependencies nor build project, so it does not support run, debugging, refactoring, linting or detecting semantic errors. For taking advantage of these features, you need switch your workspace from lightweight mode to standard mode.
+
+You can control which mode to launch with by configuring `java.server.launchMode` with below options:
+
+- `Hybrid` (default) - Firstly, a workspace is opened with lightweight mode. You will be asked whether switch to standard mode, when your source files and JDK are resolved by language server, or when an extension requires to run with standard mode. Or, you can switch by clicking server mode icon on status bar. Once a workspace is in standard mode, it cannot be switched back to lightweight mode.
```

Comment:
Feels the description is not clear.

Firstly, a workspace is opened with lightweight mode. You will be asked whether to switch to standard mode if your workspace contains Java projects that have not been imported before. If you choose "Later", it will stay in lightweight mode. And you can manually click the server mode icon on status bar to switch to standard mode.

Reply:
Corrected in the document.